ansible/inventory/env/group_vars/all.yml

```diff
@@ -78,7 +78,7 @@ single_node_es_host: "{{ groups['core-es'][0] }}"
 
 #Druid Proxy APi service
 sunbird_druid_broker_host: "http://{{ groups['raw-broker'][0] }}"
-sunbird_learner_service_url: "http://{{private_ingressgateway_ip}}/learner"
+sunbird_learner_service_url: "http://{{private_ingressgateway_ip}}/userorg"
```

Comment:
Please revert this change. This change is required from release-5.4.0 onwards.

Reply:
@AmiableAnil But when i test with release 5.3 branch itself, it was failing. If i revert back this changes, If i deploy our forked repo will it work ?

Reply:
@AmiableAnil I have reverted the changes and pushed, Please review it and merge.
But please answer to my above query as well ..